docs/brushes/AcrylicBrush.md

```diff
@@ -0,0 +1,75 @@
+---
+title: AcrylicBrush
+author: Sergio0694
+description: A composition brush that implements an acrylic effect with customizable parameters.
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, brush, backdrop, blur, win2d, composition
+---
+
+# AcrylicBrush
+
+The [AcrylicBrush](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.media.acrylicbrush) is a [Brush](https://docs.microsoft.com/uwp/api/windows.ui.xaml.media.brush) that implements an acrylic effect with customizable parameters.
+
+> [!div class="nextstepaction"]
+> [Try it in the sample app](uwpct://Brushes?sample=AcrylicBrush)
+
+## Syntax
+
+```xaml
+<Border BorderBrush="Black" BorderThickness="1" VerticalAlignment="Center" HorizontalAlignment="Center" Width="400" Height="400">
+  <Border.Background>
+    <brushes:AcrylicBrush
+        Source="Backdrop"
+        Tint="DimGray"
+        TintMix="0.25"
+        BlurAmount="20"
+        TextureUri="ms-appx:///Assets/BrushAssets/NoiseTexture.png"/>
+  </Border.Background>
+</Border>
+```
+
+## Example Image
+
+![Acrylic brush](../resources/images/Brushes/AcrylicBrush.jpg "Acrylic brush")
+
+## Properties
+
+| Property | Type | Description |
+| -- | -- | -- |
+| Source | AcrylicBackgroundSource | The source mode for the effect. |
+| BlurAmount | double | The blur amount for the effect. |
```

Comment:
We should call out that blur isn't applicable if the source is hostbackdrop?

Reply:
Fixed in https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/pull/294/commits/01dce7c0dc758c6f2695eb958a270e9549584561.